src/open_company_web/components/topic.cljs

```diff
@@ -208,7 +208,9 @@
 
 (defn topic-click [data owner options expanded selected-metric]
   (if (utils/is-mobile)
-    (mobile-topic-animation data owner options expanded)
+    (do (dis/dispatch! [:topic/toggle-expand (keyword (:section-name options))])
+        (mobile-topic-animation data owner options expanded)
+        (after 50 #(scroll-to-topic-top (om/get-ref owner "topic"))))
```

Comment:
do we really need another function for this? can't we call directly `(js/setTimeout #(scroll-to-topic-top (om/get-ref owner "topic")) 50)` @martinklepsch